core-bundle/src/EventListener/DoctrineSchema/AdjustSearchUrlLengthListener.php

```diff
@@ -48,7 +49,8 @@ public function __invoke(GenerateSchemaEventArgs $event): void
 
         // Reduce maximum length if collation is not "ascii_bin"
         if ('ascii_bin' !== $column->getPlatformOption('collation')) {
-            $bytesPerChar = 'utf8mb4' === $table->getOption('charset') ? 4 : 3;
+            $charset = $table->hasOption('charset') ? $table->getOption('charset') : 'utf8mb4';
```

Comment:
Can we safely assume `utf8mb4` as fallback value?

Reply:
It's the same in our existing `MysqlInnodbRowSizeCalculator`. /cc @ausi

Reply:
I guess we can rely on what we have configured in the skeleton config.yml.

Reply:
The default table options can be changed - and this is done quite frequently, especially on environments with older MySQL versions (see also https://docs.contao.org/manual/de/installation/systemvoraussetzungen/#mysql-mindestanforderungen).

> Or can we ask doctrine for the actually configured default instead?

It would be correct to do so, yes. However, usually the table option for the `charset` for example is unlikely to be missing. It is only missing in the functional tests as the database is set up differently there.

Reply:
So we keep the hard coded default `utf8mb4` for now as is until we see issues with it?

Reply:
Unless you see any issues with that, yes.